index.html

```diff
@@ -0,0 +1,791 @@
+                    <li class="navlinka">
+                        <span> Guides </span>
+                    </li>
+                    <li class="navlinka">
+                        <span> Books </span>
+                    </li>
+                    <li class="navlinka">
+                        <span> Events </span>
+                    </li>
+                    <li class="navlinka">
+                        <span> Memberships </span>
+                    </li>
+                    <li class="navlinka">
+                        <span> Jobs </span>
+                    </li>
+                    <li class="navlinka navlinkmore">
+                        <span> more </span>
+                    </li>
+                    <li class="sarchicon-style">
+                        <span>
+                            <i class="fa fa-search"></i><input class="searchinput-style" type="search">
+                        </span>
+                    </li>
+                </ul>
+            </nav>
+
+            <section class="title-section">
+                <div class="title-flex">
+                    <h2 class="title-h1">
+                        <span> Lorem Ipsum Lorem Ipsum </span>
+                    </h2>
+                    <h3>
+                        <span> Lorem Ipsum </span>
+                    </h3>
+                </div>
+
+                <div class="title-flex-1">
+                    <span class="title-img">
+
+                    </span>
+                    <button type="button">
+                        <span> Lorem Ipsum </span>
+                    </button>
+                </div>
+            </section>
+
+        </header>
+        <main>
+            <div class="div1 class1">
+                <div class="flexcontainer">
+                    <img class="imgmain" src="img/grey-user.png" alt="">
+                    <h3>
+                        <span> Author 1 wrote </span>
+                    </h3>
+                </div>
+                <div>
+                    <h1>
+                        <span>
+                            How to create A porsche <br>
+                            911 with sketch (part1)
+                        </span>
+                    </h1>
+                </div>
+                <div>
+
+                    <i class="far fa-comment-alt"></i>
+                    <span> Leave a comment </span>
+                </div>
+
+                <div class="flexcontainer-1">
+                    <p>
+                        <span> sketch </span>
+                    </p>
+                    <p>
+                        <span> workflow </span>
+                    </p>
+                    <p>
+                        <span> design </span>
+                    </p>
+                </div>
+
+                <div class="arrow-button">
+                    <span><i class="fas fa-arrow-right"></i> </span>
+                </div>
+            </div>
+
+            <div class="div2 class1">
+
```

Comment:
## Article tag

It uses [`article` tag](https://www.w3schools.com/tags/tag_article.asp) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/article_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/))

- Change all `<div class="div class1">` tags to `<article>` tag in HTML file

Reply:
It's done.